Source/JavaScriptCore/bytecode/UnlinkedFunctionExecutable.h

```diff
@@ -263,7 +263,7 @@ class UnlinkedFunctionExecutable final : public JSCell {
     unsigned m_isGeneratedFromCache : 1;
     unsigned m_lineCount : 31;
     unsigned m_hasCapturedVariables : 1;
-    unsigned m_unlinkedFunctionNameStart : 31;
+    unsigned m_unlinkedFunctionStart: 31;
```

Comment:
Hard to find a meaningful name for this - "expressionStart" feels wrong as it's not always an expression "declarationStart" neither for the antagonist reason. 

Also, looking at the parser code I'm wondering if this couldn't be "merged" with `m_typeProfilingStartOffset` -- Probably not a good idea to do it there but just a though.